etc/docker/dev/README.rst

```diff
@@ -54,9 +54,9 @@ To verify that everything is in order, you can now either run the full unit test
 Alternatively, you can bootstrap the test environment once with the `-i` option and then selectively or repeatedly run test case modules, test case groups, or even single test cases, for example::
 
     tools/run_tests_docker.sh -i
-    tools/pytest.sh test_replica.py
-    tools/pytest.sh -vvv test_replica.py::TestReplicaCore
-    tools/pytest.sh -vvv --full-trace test_replica.py::TestReplicaCore::test_delete_replicas_from_datasets
+    tools/pytest.sh tests/test_replica.py
```

Comment:
Just a slight note, for the donkeyrider, we will have to probably specify the values for `--filter-tests` in the tests/<filter> format ;) 
https://github.com/rucio/rucio/blob/81bacae53dbe2486bcff87d55feb0dc65daa7ac0/tools/test/donkeyrider.py#L126C9-L126C9 

Not sure what would be the correct place to document it. Maybe the `donkeyrider.py` itself?

Reply:
I'm not sure I understand correctly the issue. Is it just about the paths in filter-tests changing ?